backend/webdav/webdav.go

```diff
@@ -67,14 +69,17 @@ func init() {
 				Help:  "Owncloud",
 			}, {
 				Value: "sharepoint",
-				Help:  "Sharepoint",
+				Help:  "Sharepoint Online, authenticated by Microsoft OneDrive account.",
```

Comment:
Isn't it just "Microsoft account"? In addition, I suggest mentioning the cookie-based authentication process here (just as NTLM is mentioned below), since this is a clear technical identifier independent from MS parlance.

Reply:
I don't mind to replace "Microsoft OneDrive account" by "Microsoft account".
In the same time I believe most users don't care about cookies and propose to keep things simple and left "cookies" out.
@Cnly please confirm

Reply:
fixed by commit https://github.com/rclone/rclone/pull/2921/commits/841bf20207699f3a35a0b009b549e33da40277cc